native/src/java_wrap.cpp

```diff
@@ -1519,6 +1642,52 @@ extern "C" JNIEXPORT jobjectArray JNICALL Java_net_osmand_NativeLibrary_nativeRo
 	c->basemap = basemap;
 	c->setConditionalTime(c->config->routeCalculationTime);
 	c->publicTransport = ienv->GetBooleanField(jCtx, jfield_RoutingContext_publicTransport);
+	ienv->DeleteLocalRef(jRouteConfig);
+	return c;
+}
+
+extern "C" JNIEXPORT jobject JNICALL Java_net_osmand_NativeLibrary_searchGpxRoute(JNIEnv* ienv, jobject obj,
```

Comment:
where is pull request for Java ? it should be properly documented on the java side which methods are related to jni calls